app/actions/index.js

```diff
@@ -18,6 +18,23 @@ export const removeDataFeed = ({ address }) => dispatch => {
   })
 }
 
+export const fetchDataFeedLatestResult = ({ address }) => dispatch => {
```

Comment:
same with `address` here and in then function below .. `dataFeedAddress` would be more descriptive, since we have a lot of addresses in the dapp that these could possibly refer to